.github/workflows/lint-and-tests.yml

```diff
@@ -202,7 +202,7 @@ jobs:
             startsWith(github.event.pull_request.head.ref, 'dependabot/') == false &&
             github.event.pull_request.head.ref != 'chore/crowdin')
         # sha reference has no stable git tag reference or URL. see https://github.com/chromaui/chromatic-cli/issues/797
-        uses: chromaui/action@30b6228aa809059d46219e0f556752e8672a7e26
+        uses: chromaui/action@c93e0bc3a63aa176e14a75b61a31847cbfdd341c
```

Comment:
BTW, I double checked https://github.com/marketplace/actions/publish-to-chromatic is a verified publisher, meaning we can upgrade safely without having to allow-list individual shas - updated the actions settings accordingly